tests/integration/utils/fixture.ts

```diff
@@ -169,6 +169,12 @@ export async function setupFixtureTests(
       }
     })
 
+    afterEach<FixtureTestContext>((context) => {
+      if (devServer && context.meta.result?.state === 'fail') {
+        console.log(devServer.output)
```

Comment:
This is useful for troubleshooting failures in the CI. When the test fails, we print the stdout we've accumulated.